packages/core/src/api/apis/default/AlertApiForwarder.ts

```diff
@@ -0,0 +1,37 @@
+import { AlertApi, AlertContext, AlertMessage } from '../../../';
+import { Color } from '@material-ui/lab';
+
+type SubscriberFunc = (error: string) => void;
+type Unsubscribe = () => void;
+
+export class AlertApiForwarder implements AlertApi {
+  private readonly subscribers = new Set<SubscriberFunc>();
+  severity: Color = 'info';
+
+  constructor(errorApi) {
```

Comment:
There's a bit of `any` going on here, the errorApi doesn't actually have a subscribe method?

Reply:
it does but yea i can type this

Reply:
ohh i see what you mean. i feel like i should just add subscribe into the error definition?  was there a reason it wasn't in there to begin with?

Reply:
if we wanted to rip out alerting for example, we'd have to update all the other apis that are calling it vs just removing its implementation